packages/flame/example/lib/main.dart

```diff
@@ -80,6 +67,7 @@ class Square extends PositionComponent with Tappable {
   @override
   bool onTapUp(TapUpInfo info) {
     removeFromParent();
+    info.handled = true;
```

Comment:
Why was this needes?

Reply:
because returning true no longer sets it automatically

Reply:
But why does it need to be set?

Reply:
so the check to avoid adding a new square works

Reply:
Ah, but just return false instead to stop the propagation, and no check is needed.

Reply:
let me try

Reply:
I thought `true` meant handled

Reply:
doesn't work

Reply:
`true` means continue to propagate, but it always ends up on the game it seems like? I wonder if that is a bug...
You can check `info.continuePropagation` on the game, but then it's cleaner to do lile you did before.

Reply:
Yeah, it must have always been like that, since that is the entry point, I'll merge this